lib/src/main/java/uk/gov/di/ipv/cri/address/library/domain/SessionRequest.java

```diff
@@ -0,0 +1,75 @@
+package uk.gov.di.ipv.cri.address.library.domain;
+
+import com.fasterxml.jackson.annotation.JsonAlias;
+
+import java.net.URI;
+import java.util.StringJoiner;
+
+public class SessionRequest {
+
+    @JsonAlias("response_type")
```

Comment:
Is there a reason to use JsonAlias over JsonProperty?
JsonAlias does not work with Enums, if we wanted to use any in future

Reply:
This allows us to alias the request params from the [OAuth authorization code grant request](https://datatracker.ietf.org/doc/html/rfc6749#section-4.1.3) i.e. we receive `response_type` as a request param to our frontend, which maps these params to a json payload to the /session API.